src/main/java/ua/ostapenko/hw10/Recovery.java

```diff
@@ -0,0 +1,17 @@
+package ua.ostapenko.hw10;
+
+interface Recovery {
```

Comment:
define what interface you need here and remove second one

Reply:
I made two interfaces with different realization, and the second one was commented. And I want to know which one the better decision?